src/fragments/lib/storage/js/download.mdx

```diff
@@ -137,11 +139,19 @@ You can use `expires` option to limit the availability of your URLs. This config
 await Storage.get('filename.txt', { expires: 60 });
 ```
 
+### Check existence of file
```

Comment:
Is there a reason why we are calling this out again in the guide?

Reply:
+1, I think the change at the top of the page is adequate

Reply:
Updated. Thanks.